src/qibo_cloud_backends/qibo_client.py

```diff
@@ -18,6 +20,7 @@ class QiboClientBackend(NumpyBackend):
 
     def __init__(self, token, provider=None, platform=None):
         super().__init__()
+        os.environ.setdefault("QRCCLUSTER_PORT", "8080")
```

Comment:
That's also fine, for the time being. But the proper place will be:
https://github.com/qiboteam/qibo-client/blob/4e70c049bb66475d9af468fe3e9611d75fd5e5fb/src/qibo_client/tii.py#L6
since providers are defined there.

Reply:
Ah ok I see, makes sense. By the way, for some reason if you set the env variable inside the backend it won't be set correctly inside of the test, but I am not sure why...

Reply:
I'm now checkingnit

Reply:
Error found: the environment is probed top-level, both in `qibo-cloud-backends` and `qibo-client`, thus it is determined at `import` time, and insensitive to updates.

This is a problem in different ways, I will file one issue per repo describing it. The hotfix is in c922218, and just reverts what you did.